sdk/core/azure-core/src/http/http.cpp

```diff
@@ -26,6 +26,7 @@ const HttpMethod HttpMethod::Post("POST");
 const HttpMethod HttpMethod::Put("PUT");
 const HttpMethod HttpMethod::Delete("DELETE");
 const HttpMethod HttpMethod::Patch("PATCH");
+const HttpMethod HttpMethod::Options("OPTIONS");
```

Comment:
Add entry to Core's README.md, under Other Changes
``- Added `OPTIONS` HTTP method.``